esp-hal/src/interrupt/software.rs

```diff
@@ -16,12 +16,24 @@
 //!
 //! // Take the interrupt you want to use.
 //! let mut int0 = sw_ints.software_interrupt0;
+//! let mut int1 = sw_ints.software_interrupt1;
```

Comment:
This is seriously unnecessary.

Reply:
I rather agree with that. Initially I wanted to leave it as an example of using multiple interrupts, but in fact user should be able to intuitively realize that they should just copy-paste what is done with one interrupt.